Xamarin.Essentials/Xamarin.Essentials.csproj

```diff
@@ -138,6 +138,8 @@
       <HintPath>../Libs/gdk-sharp.dll</HintPath>
       <Private>false</Private>
     </Reference>
+    <Compile Include="**\*.netstandard.cs" />
+    <Compile Include="**\*.netstandard.*.cs" />
```

Comment:
@webwarrior-ws this looks scary, compiling netstandard files in the net461 profile?

Reply:
That's how it works. There are no separate .net461.cs files.

Reply:
but the stable branch didn't do it this way, look at the commit there again please

Reply:
But it did:
https://github.com/nblockchain/DotNetEssentials/blob/stable/Xamarin.Essentials/Xamarin.Essentials.csproj
lines 128-129

Reply:
holy shit, you're right; and the commit that introduces Clipboard support also changes Connectivity, so fucked up. We need to rethink this.